jekyll/_cci2/configuration-reference.md

```diff
@@ -144,6 +144,7 @@ docker | Y <sup>(1)</sup> | List | Options for [docker executor](#docker)
 machine | Y <sup>(1)</sup> | Map | Options for [machine executor](#machine)
 macos | Y <sup>(1)</sup> | Map | Options for [macOS executor](#macos)
 shell | N | String | Shell to use for execution command in all steps. Can be overridden by `shell` in each step (default: See [Default Shell Options](#default-shell-options))
+parameters | Y | List | A list of parameters that need to be passed as part of the request when performing this operation
```

Comment:
I don't think the `parameters` are a `required` key 🤔

Reply:
Fixed